README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.com/IBM/the-mesh-for-data.svg?branch=master)](https://travis-ci.com/IBM/the-mesh-for-data "Travis")
 
```

Comment:
I think that the empty line should be removed

Reply:
i agree but not sure how to edit from github WebUI (via my mobile)

Reply:
Fixed